.circleci/config.yml

```diff
@@ -1760,6 +1763,29 @@ jobs:
             name: "Open a PR to create release notes into docs repository"
             working_directory: "./release"
             command: npm run zx -- ci-steps/doc-new-release-notes.mjs --version=${VERSION}
+        - when:
+              condition:
+                  not: << pipeline.parameters.dry_run>>
+              steps:
+                  - run:
+                        name: Get RELEASE_NOTES_PR_URL
+                        command: |
+                            echo "export RELEASE_NOTES_PR_URL=$(cat /tmp/releaseNotesPrUrl.txt)" >> $BASH_ENV
+                  - slack/notify:
+                        channel: C02NGT20S4W
```

Comment:
Does it use a new channel? Or the one we are using for all release alerts?

Reply:
[#gravitee-release-alerts](https://graviteeio.slack.com/archives/C02NGT20S4W)

Reply:
I don't think a new channel is necessary 🤷‍♂️

Reply:
seen with the @tech_writers_team 🚀 
> the release channel should be the best place to ping us from yes

p.s : new slack alias 🎸